modules/scaleway/versions.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    scaleway = {
+      source = "scaleway/scaleway"
+      version = ">= 1.17.0"
```

Comment:
version = "~> 1.0"

Reply:
there is some breaking changes starting from `1.11`.

Reply:
Then they should have make it 2.0 :p

Reply:
Not my provider :P